internal/generate/testdata/non-standard-layout/config/crds-with-core-types/foo.example.com_deployment_crd.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We need to migrate this test for the new layout but I think it is fine to do it after in a follow-up.

Reply:
> We need to migrate this test for the new layout but I think it is fine to do it after in a follow-up.

Yes, i will do.